bitcoin/src/psbt/map/global.rs

```diff
@@ -149,7 +149,7 @@ impl PartiallySignedTransaction {
                                 }
                                 let derivation = DerivationPath::from(path);
                                 // Keys, according to BIP-174, must be unique
-                                if xpub_map.insert(xpub, (Fingerprint::from(&fingerprint[..]), derivation)).is_some() {
+                                if xpub_map.insert(xpub, (Fingerprint::from(fingerprint), derivation)).is_some() {
                                     return Err(encode::Error::ParseFailed("Repeated global xpub key"))
```

Comment:
I think there are two missing tests that need to be checked

Reply:
Unrelated but which ones? Why do you think?